File: Composicion_y_Agregacion/Ejercicio1/Java/Main.java
package Composicion_y_Agregacion.Ejercicio1.Java;

public class Main {
    public static void main(String[] args) {
    System.out.println("-------Casa:------");
    Casa micasa = new Casa ("Calle Flores #123", 5); 
    Habitacion h1 = new Habitacion ("Sala", 20.0); 
    Habitacion h2 = new Habitacion ("Cocina", 12.5); 
    Habitacion h3 = new Habitacion ("Dormitorio", 15.0); 
    Habitacion h4 = new Habitacion ("Baño", 6.0);

    micasa.agregar_habitacion (h1);
    micasa.agregar_habitacion (h2);
    micasa.agregar_habitacion (h3);
    micasa.agregar_habitacion (h4);
    micasa.mostrar_casa();
    }
}

File: Composicion_y_Agregacion/Ejercicio1/Python/Main.py

from Casa import Casa
from Habitacion import Habitacion

if __name__ == "__main__":
    print("-------Casa: -------")
    mi_casa = Casa("Calle Flores #123", 5)
    
    h1 = Habitacion("Sala", 20.0)
    h2 = Habitacion("Cocina", 12.5)
    h3 = Habitacion("Dormitorio", 15.0)
    h4 = Habitacion("Baño", 6.0)
    
    mi_casa.agregar_habitacion(h1)
    mi_casa.agregar_habitacion(h2)
    mi_casa.agregar_habitacion(h3)
    mi_casa.agregar_habitacion(h4)
    mi_casa.mostrar_casa()
